spec/models/keyword_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Keyword, type: :model do
+  describe 'Init' do
+    let(:keyword) { Fabricate(:keyword) }
+
+    it 'has name' do
+      expect(keyword.name).not_to be_nil
+    end
```

Comment:
I agree with you, this test is redundant of the init functionality of the object but I would like to keep it here so that the `keyword` model has some test that I can build on in the future features. I think of it as a starting point for other tests for this model.

Taking you suggestion, I will update the test to not use Fabricator for the `expose` test, as we are testing the `init` function and also the `read` permission.

Reply:
Ok. 

Small note: `read permission` => It's usually called `visibility` (attributes visibility, method visibility, ... Either public, protected or private).

Let's keep `permission` for requests params and authorization ;-)